src/finetuning/config/experiment.py

```diff
@@ -47,9 +46,9 @@ class Overrides(BaseConfig):
     lr_batch_size_scaling: Literal["none", "sqrt", "linear"] = Field(
         default="none",
         description=dedent(
-            """Scales the learning rate in the training_args by a factor derived from the total training batch size.
-            'none': No scaling.
-            'sqrt': Multiplies learning rate by square root of batch size (a classic scaling rule).
+            """Scales the learning rate in the training_args by a factor derived from the total training batch size. \
+            'none': No scaling. \
+            'sqrt': Multiplies learning rate by square root of batch size (a classic scaling rule). \
```

Comment:
Why is this needed? Shouldn't the triple quote string handle line breaks already?